src/codegen/codegen_cpp_visitor.cpp

```diff
@@ -895,6 +895,9 @@ void CodegenCppVisitor::setup(const Program& node) {
 
     update_index_semantics();
     rename_function_arguments();
+
+    /// TODO: Calculate \c ppvar_count properly
+    info.ppvar_count = int_variables_size() + info.emit_cvode;
```

Comment:
@iomaganaris : just to clarify this for future handling/development: on coreneuron side, `ppvar` typically represents number of `int` variables. Changing `int_variables_size()` to consider additional variables of `info.emit_cvode` breaks many things and hence this is todo?

Reply:
Good point. I think the proper solution is to add the `cvodeieq` semantic to the `info.semantics`. However at the moment I/Michael are not sure how this is set